test/http-client.spec.js

```diff
@@ -535,14 +535,11 @@ describe('http client', () => {
     });
 
     it('should reject when aborted', (done) => {
-      // TODO: Enable and fix this test when
-      // https://github.com/jasmine/jasmine-ajax/issues/111
-      // will be fixed
       var client = new HttpClient()
         .configure(x => x.withBaseUrl(baseUrl));
 
       var promise = client.send(new HttpRequestMessage('GET', 'some/cool/path'));
-      promise.catch((response, reason) => {
+      promise.then(() => { done(); }, (response, reason) => {
```

Comment:
I don't think this is right. It looks like this test will now pass if the promise resolves instead of rejects. We should verify that it doesn't resolve successfully.